rush-plugins/rush-resolver-cache-plugin/src/afterInstallAsync.ts

```diff
@@ -0,0 +1,357 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import { createHash } from 'node:crypto';
+import * as path from 'node:path';
+
+import type {
+  RushSession,
+  RushConfiguration,
+  RushConfigurationProject,
+  ILogger,
+  LookupByPath,
+  Subspace
+} from '@rushstack/rush-sdk';
+import type {
+  ISerializedResolveContext,
+  IResolverCacheFile
+} from '@rushstack/webpack-workspace-resolve-plugin';
+
+import { Async, FileSystem, PnpmShrinkwrapFile } from './externals';
+
+const MAX_LENGTH_WITHOUT_HASH: number = 120 - 26 - 1;
+const BASE32: string[] = 'abcdefghijklmnopqrstuvwxyz234567'.split('');
```

Comment:
No `1`?

Reply:
https://datatracker.ietf.org/doc/html/rfc4648#page-8

Base32 picks its characters specifically to avoid glyphs that are too similar.